invenio_oauthclient/handlers/rest.py

```diff
@@ -39,6 +39,8 @@ def response_handler_postmessage(remote, url, payload=None):
 
 def default_response_handler(remote, url, payload=None):
     """Default response handler."""
+    default_handler = current_app.config.get("OAUTHCLIENT_REST_DEFAULT_RESPONSE_HANDLER")
+    return obj_or_import_string(default_handler)(remote, url, payload)
```

Comment:
:/ this return is in the middle of the function

Reply:
Argh, I removed one line too much. Sorry I'm tired today.